.github/workflows/docker.yml

```diff
@@ -31,18 +33,26 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Compute labels
+        id: labels
+        run: |
+          REF=$(echo "${{ github.ref_name }}" | sed 's/\//_/g')
+          echo "ref=$REF" >> $GITHUB_OUTPUT
+          DATE=$(echo "${{ github.event.repository.updated_at }}" | head -c 10)
+          echo "date=$DATE" >> $GITHUB_OUTPUT
+          echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+
       - name: Build image and push to GitHub Container Registry
         uses: docker/build-push-action@v3
         with:
           context: .
           tags: |
             ghcr.io/chainsafe/forest:latest
+            ghcr.io/chainsafe/forest:${{ steps.labels.outputs.date }}-${{ steps.labels.outputs.sha_short }}
+            ghcr.io/chainsafe/forest:${{ steps.labels.outputs.ref }}
```

Comment:
Given that those tags will be pushed only on `main`, we will end up with `latest` and `main` on each commit, right? I am not sure we want to keep `main` in this case. Not a big deal though.

Reply:
I modified the code such that `ref` is either the tag name or `latest`. A bit of a hack but I don't see how to clean it up.

Reply:
Looks good, thanks.